### ISSUE TITLE
I pray and hope EtherealSX2 becomes the best PS2 Game Emulator in the world

### ISSUE THREAD

**Issue by Ottorock** [Trixarian/EtherealSX2#4]
On Android & Others it will take time/effort through four , five Or six years it’ll be worth the wait I know the Real PS2 has a lot of games 4,000 (that’s why I said what I said)😭💔💯 also I have friends that can help you with your donation one thing can I show my friends your work if that’s okay with you

**Comment by Mem500-cpu** [comment]
> On Android & Others it will take time/effort through four , five Or six years it’ll be worth the wait I know the Real PS2 has a lot of games 4,000 (that’s why I said what I said)😭💔💯 also I have friends that can help you with your donation

Yes it will quite good ps2 emulator come out just going take awhile sure modern pcsx2 ver slight more heavy. Think going wait between 1 to 2 year and once happen they be on track for good emulation hopefully.
